wordpress/wp-content/plugins/cds-base/classes/Modules/Contact/ContactForm.php

```diff
@@ -41,227 +60,198 @@ public function render($atts, $content = null): string
-                            name="sending-integration"
-                    ></textarea>
-                    </div>
-
-                    <div class="buttons">
-                        <button class="gc-button gc-button" type="submit" id="submit">
-                            <?php _e('Submit', 'cds-snc'); ?>
-                        </button>
-                    </div>
-                </form>
-                
-                <?php else : ?>
-                <form id="contact-form" method="POST" action="/wp-json/contact/v1/process">
-                    <?php // echo $heading; ?>
-                    <?php wp_nonce_field(
-                        'contact_form_nonce_action',
-                        'contact',
+                    <p><?php _e('We use this information to improve GC Articles.', 'cds-snc');?></p>
+                    
+                    <div class="focus-group" style="margin-bottom: 1.75rem">
+                    <?php $this->checkBox(
+                        'usage[]',
+                        'Blog.',
+                        __('Blog.', 'cds-snc'),
+                    ); ?>
+                    <?php $this->checkBox(
+                        'usage[]',
+                        'Newsletter archive with emailing to a subscriber list.',
+                        __('Newsletter archive with emailing to a subscriber list.', 'cds-snc'),
+                    ); ?>
+                    <?php $this->checkBox(
+                        'usage[]',
+                        'Website.',
+                        __('Website.', 'cds-snc'),
+                    ); ?>
+                    <?php $this->checkBox(
+                        'usage[]',
+                        'Internal website.',
+                        __('Internal website.', 'cds-snc'),
+                    ); ?>
+                    <?php $this->checkBox(
+                        'usage[]',
+                        'Something else.',
+                        __('Something else.', 'cds-snc'),
                     ); ?>
-                    <input type="hidden" name="contact-type" value="<?php echo $contactType; ?>"/>
-                    <div class="focus-group">
-                        <label class="gc-label required" for="email" id="email-label">
-                            <?php _e('Email', 'cds-snc'); ?>
-                        </label>
-                        <input type="email" class="gc-input-text" id="email" required autocomplete="email"
-                               name="email" value="">
                     </div>
-                    <label data-testid="description" class="gc-label required" id="contact-label" for="message">
-                        <?php echo $label; ?>
+                    <input 
+                        type="text" 
+                        class="gc-input-text" 
+                        id="usage-other" 
+                        name="usage-other" 
+                        value=""
+                    />
```

Comment:
These are getting flagged in axe for not having labels. We could create a visually hidden label that says "Other usage" or something.